Robustly handle config file syntax errors [PREVIEW]

nicoe opened the pull request.

AdrienBenduc commented:
https://bugs.tryton.org/issue6731

AdrienBenduc commented:
Up

AdrienBenduc commented:
@JCavallo  ca fonctionnait,
Cédric a refactore le code
http://hg.tryton.org/tryton/rev/438af1da332d/

jmousset commented:
merged with https://github.com/coopengo/tryton/pull/26

JCavallo committed:
[tryton] Remove leftover of expander removal on M2M [CUSTOM]

JCavallo committed:
[tryton] Remove left over of signal removal [CUSTOM]

JCavallo committed:
[tryton] Remove check on tab_domain [CUSTOM]

JCavallo committed:
[tryton] Correctly initialize EditableTreeView [CUSTOM]

JCavallo committed:
[tryton] Ensure that click on button in list-form mode selects first the line [PREVIEW] (#216)

Fix #23314

JCavallo committed:
[tryton] Iteration on mixt widget [CUSTOM]

JCavallo committed:
[modules/account] account: Auto reconcile after move is posted [PREVIEW]

Fix PROCK-186

JCavallo committed:
[tryton] Accept any arguments for _focus_out [CUSTOM] (#215)

Fix #24366

JCavallo committed:
[coopengo_modules/electronic_signature]
electronic_signature: Add a timeout to signature service calls

JCavallo committed:
[coopengo_modules/debug] Allow to autoprofile wizards

JCavallo committed:
[coopengo_modules/electronic_signature] electronic_signature: set timeout in conf returned by get_credentials (#23)

Fix #PROCK-283

JCavallo committed:
[modules/account_dunning] account_dunning: Fix a translation

Fix #PJAZZ-671

JCavallo committed:
[sao] Add .gitignore [CUSTOM]

JCavallo committed:
[modules/account] account: Add condition to check if a line is reconciled before attempting to reconcile it when posting a move [CUSTOM]

Fix #PJAZZ-703

When cancelling a move through the move cancellation wizard, we reconcile the lines and then post the generated cancellation move. However, when posting the move, we currently try to reconcile lines on which there is no credit/debit, which results in an error because these lines have already been reconciled in the previous step.

JCavallo committed:
[modules/party] party: Add missing depends [CUSTOM] (#6)

Fix #PJAZZ-735

JCavallo committed:
[sao] add generated translations to gitignore [CUSTOM]

JCavallo committed:
[coopengo_modules/debug] Do not crash on wrong login

JCavallo committed:
[sao] Style: fix #4514 [CUSTOM]

JCavallo committed:
[sao] Make it usable for coog [CUSTOM]

fixed bugs and implemented coog specific features

Fix #3643
http://rietveld.coopengo.com/20970001/

JCavallo committed:
[sao] add files [CUSTOM]

JCavallo committed:
[sao] Open menu entries with a doubleclick [CUSTOM]

dbclick on menu
dbclick on treeviews
hidden checkboxes in trees
fix #4830

JCavallo committed:
[sao] secure group.remove() in _set_value() of O2M [CUSTOM]

call group.remove() only when necessary
compare for undefined when compared to null
fix #4752

JCavallo committed:
[sao] set a "min-height" on textareas [CUSTOM]

fix #4846

JCavallo committed:
[sao] Change icons [CUSTOM]

fix #4854

JCavallo committed:
[sao] quickfix on DateTimePicker [CUSTOM]

fix #4853

JCavallo committed:
[sao] Fix menu icon width [CUSTOM]

JCavallo committed:
[sao] Rename theme [CUSTOM]

JCavallo committed:
[sao] Report dev on toggle for processes from gtk client [CUSTOM]

JCavallo committed:
[sao] Make nav-bar btn glyphicon blue [CUSTOM]

JCavallo committed:
[sao] Fix global search icons [CUSTOM]

JCavallo committed:
[sao] Add package-lock.json to gitignore [CUSTOM]

JCavallo committed:
[sao] Start to define the palette in one file [CUSTOM]

JCavallo committed:
[sao] Get dist out of source control [CUSTOM]

Fix #8180

JCavallo committed:
[sao] Handle inputs.less [CUSTOM]

JCavallo committed:
[sao] Refactore global-header and main-screen [CUSTOM]

JCavallo committed:
[sao] Handle main-tabs.less [CUSTOM]

JCavallo committed:
[sao] Handle panel.less [CUSTOM]

JCavallo committed:
[sao] Handle tabs.less [CUSTOM]

JCavallo committed:
[sao] Handle tree.less [CUSTOM]

JCavallo committed:
[sao] Order by color, more comments [CUSTOM]

JCavallo committed:
[sao] Add comments on tree.less [CUSTOM]

JCavallo committed:
[sao] Remove locale files [CUSTOM]

JCavallo committed:
[sao] Fix comment [CUSTOM]

JCavallo committed:
[sao] Selection in tree view is blue, fix #8184 , fix #8183 [CUSTOM]

JCavallo committed:
[sao] Fix comment, fix oneline_inputs.less [CUSTOM]

JCavallo committed:
[sao] define coog palette according to chart [CUSTOM]

JCavallo committed:
[sao] Remove deprecated comment [CUSTOM]

JCavallo committed:
[sao] set main-screen-font-color to black [CUSTOM]

JCavallo committed:
[sao] Begin setting of colors according to #8287 [CUSTOM]

JCavallo committed:
[sao] Rm generated translations from git (#6) [CUSTOM]

Fix #8180

JCavallo committed:
[sao] Reduce menu width for big screens [CUSTOM]

Fix #8296

JCavallo committed:
[sao] Style [CUSTOM]

JCavallo committed:
[sao] Pierre's review [CUSTOM]

JCavallo committed:
[sao] Pîerre's review [CUSTOM]

JCavallo committed:
[sao] Iteration on theme [CUSTOM]

Fix #8297

JCavallo committed:
[sao] Tabs [CUSTOM]

JCavallo committed:
[sao] Tabs size [CUSTOM]

JCavallo committed:
[sao] Correctly implement no_command attribute on Dict widget [CUSTOM]

JCavallo committed:
[sao] Iteration on theme (#15) [CUSTOM]

Fix #4852

JCavallo committed:
[sao] Remove 1 px to tree view to hide default scroll bar (#16) [CUSTOM]

Fix #8630

JCavallo committed:
[sao] Use coog icons for ged toolbar (#22) [CUSTOM]

Fix #4852

JCavallo committed:
[sao] Add temporary fix for multi_mixed_view crash [CUSTOM]

JCavallo committed:
[sao] Modify badge (#24) [CUSTOM]

JCavallo committed:
[sao] Increase tree spaces, add space after list icons (#23) [CUSTOM]

JCavallo committed:
[sao] Backport refactoring of tree views [CUSTOM]

hide .selection-state
cherry-pick be33c6ab
cherry pick http://hg.tryton.org/sao/rev/cf989f7a944e/
cherry pick cd672b095af690c0a38123725b8a71c00fed5344

Fix #7226 #8474

JCavallo committed:
[sao] Add width % on canvas (#20) [CUSTOM]

JCavallo committed:
[sao] Add white color for link (#25) [CUSTOM]

JCavallo committed:
[sao] Remove link when line is not selected (#26) [CUSTOM]

* remove link when line is not selected

* Typo & CHANGELOG

JCavallo committed:
[sao] Extra data: display, and readonly (#27) [CUSTOM]

JCavallo committed:
[sao] Tree: do not call focus if widget does not define it [CUSTOM]

Fix #8720

JCavallo committed:
[sao] Theme: Fix size of extra button on toolbar [CUSTOM]

JCavallo committed:
[sao] Add changelog and comment [CUSTOM]

JCavallo committed:
[sao] Tree: prevent crash on prevalidate of m2m [CUSTOM]

Fix #8810

JCavallo committed:
[sao] Build: clean scripts (#29) [CUSTOM]

JCavallo committed:
[sao] Remove modal link (#33) [CUSTOM]

JCavallo committed:
[sao] Launch both client action and action

JCavallo committed:
[sao] Fix width of selection input (#34) [CUSTOM]

JCavallo committed:
[sao] Add scrollbar on modal (#32) [CUSTOM]

JCavallo committed:
[sao] Bug/8817 (#35) [CUSTOM]

Fix #8817

JCavallo committed:
[sao] Remove checkbox for always ignoring warnings [CUSTOM]

Fix #9035

JCavallo committed:
[sao] Rename the title used in the browser [CUSTOM]

Fix #8785

JCavallo committed:
[sao] Remove console.log [CUSTOM]

JCavallo committed:
[sao] Use CodeMirror to handle SourceView [CUSTOM]

JCavallo committed:
[sao] Add linter [CUSTOM]

JCavallo committed:
[sao] Add some readonly theme [CUSTOM]

JCavallo committed:
[sao] Reduce size below code editor [CUSTOM]

JCavallo committed:
[sao] Only allow open link when using ctrl key (#46) [CUSTOM]

Fix #8756

JCavallo committed:
[sao] Comment linting example [CUSTOM]

JCavallo committed:
[sao] CSS: boolean input to the left (#47) [CUSTOM]

Fix #9263

JCavallo committed:
[sao] Load CodeMirror inside docker (#51) [CUSTOM]

Fix #9504

JCavallo committed:
[sao] Only set width 100% of input-form inside dict (#48) [CUSTOM]

Fix #9446

JCavallo committed:
[sao] Fix populate_funcs [CUSTOM]

JCavallo committed:
[sao] Add linting service support [CUSTOM]

JCavallo committed:
[sao] Set default color on btn-primary (#53) [CUSTOM]

Fix #9678

JCavallo committed:
[sao] Remove color change on dropdown link when hover (#52) [CUSTOM]

Fix #9680

JCavallo committed:
[sao] Add min-width of 12em to m2o input [CUSTOM]

Fix #9532

JCavallo committed:
[sao] Fix commit 7dd58cf55dae4609730ba26a9d7fce092cb68cbf [CUSTOM]

Fix #9678

JCavallo committed:
[sao] Allow list view when Ctrl+click (#64) [CUSTOM]

Fix #10098

JCavallo committed:
[sao] Autorefresh modal codeMirror using addon (#65) [CUSTOM]

Fix #9909

JCavallo committed:
[sao] Clean session if local session is wrong and we got an ir.session error (#61) [CUSTOM]

Fix #10127

JCavallo committed:
[sao] Iteration following tests [CUSTOM]

JCavallo committed:
[sao] Fix build permissions [CUSTOM]

JCavallo committed:
[sao] Fix dockerfile & fix icons size [CUSTOM]

JCavallo committed:
[sao] Properly handle no_command [CUSTOM]

JCavallo committed:
[sao] Fix multiple view promise synchronization [CUSTOM]

JCavallo committed:
[sao] Ensure every field is loaded when selecting a node in a multimodel+group widget [CUSTOM]

JCavallo committed:
[sao] Set the subview record of mixed-view before displaying [CUSTOM]

Failure to do so results in a widget that is not displayed, redering the
parent td containing it collapsable finaly resulting in an pile of poo
on the display of the form

JCavallo committed:
[sao] Correctly schedule subviews display [CUSTOM]

JCavallo committed:
[sao] Call container resize when displaying the widget [CUSTOM]

JCavallo committed:
[sao] Fix rule_engine [CUSTOM]

JCavallo committed:
[sao] Remove offcanvas from docker image build [CUSTOM]

JCavallo committed:
[sao] Adaptation of the sao icons [CUSTOM]

Fix #11200

JCavallo committed:
[sao] Adaptation of the sao icons [CUSTOM]

Fix #11200

JCavallo committed:
[sao] Add material design pencil , use it as m2o edit icon [CUSTOM]

The icon is from : https://material.io/tools/icons/?icon=edit&style=baseline

Fix #11425

JCavallo committed:
[sao] Update Sao.View.Form.Source to use record and field accessors [CUSTOM]

JCavallo committed:
[sao] Always test against undefined and null in group_sync [CUSTOM]

Fix #11637

JCavallo committed:
[sao] Fix get method Reference field [CUSTOM]

temporary fix for #14945

JCavallo committed:
[sao] Fix glyphicon font load (#92) [CUSTOM]

Fix #13453

JCavallo committed:
[sao] Add never_modified flag (#95) [CUSTOM]

JCavallo committed:
[sao] Correctly set value from on_change calls [CUSTOM]

JCavallo committed:
[sao] Better looking dictionary forms [CUSTOM]

JCavallo committed:
[sao] Restore old icons, still used coog side [CUSTOM]

JCavallo committed:
[sao] Fix width of menu issue [CUSTOM]

JCavallo committed:
[sao] Imlpement dynamic domain evaluation [CUSTOM]

JCavallo committed:
[sao] Remove handling of 'always' in UserWarning dialogs [CUSTOM]

JCavallo committed:
[sao] Add missing Sortable to depencies [CUSTOM]

Fix #17889

JCavallo committed:
[sao] Handle table headers colour [CUSTOM]

JCavallo committed:
[sao] Set readonly both inputs of numeric dict fields [CUSTOM]

JCavallo committed:
[sao] Fix recursive context and wrong handling of tab_domain in search_count (#113) [CUSTOM]

Fix #19009

JCavallo committed:
[sao] Fix wrong super call from d01cb6bc7577306b (#123) [CUSTOM]

JCavallo committed:
[sao] Add option to remove leading and trailing whitespaces from fields.Char (#121) [CUSTOM]

fix #19369

JCavallo committed:
[sao] Replace Sortable.min.js by Sortable.js in index.html [CUSTOM]

JCavallo committed:
[sao] Update build script post merge (#124) [CUSTOM]

JCavallo committed:
[sao] Use coog theme, set icon-background-color to white [CUSTOM]

JCavallo committed:
[sao] Fix input size for date and checkbox [CUSTOM]

JCavallo committed:
[sao] Fix headers [CUSTOM]

JCavallo committed:
[sao] Fix hover issue [CUSTOM]

JCavallo committed:
[sao] Remove box-shadow around btn [CUSTOM]

JCavallo committed:
[sao] Remove box-shadow around dict [CUSTOM]

JCavallo committed:
[sao] Enforce fullcalendar version [CUSTOM]

JCavallo committed:
[sao] Restore boxes aournd o2m [CUSTOM]

JCavallo committed:
[sao] Activate double click on editable treeviews (#134) [CUSTOM]

Fix #22200

JCavallo committed:
[sao] Propagate changed status to parent on instance button activation (#131) [CUSTOM]

Fix #21762

JCavallo committed:
[sao] Add missing semi colon [CUSTOM]

JCavallo committed:
[sao] Return rec_name promise when available (#129) [CUSTOM]

Fix #21445

JCavallo committed:
[sao] Add database timeout parameter on RPC calls (#140) [PREVIEW]

Fix #22550

JCavallo committed:
[sao] Fix build/ep script for boostrap fonts [CUSTOM]

Fix #23106

JCavallo committed:
[sao] Fix build [CUSTOM]

JCavallo committed:
[sao] Ensure that click on button in list-form mode selects first the line [PREVIEW]

JCavallo committed:
[sao] Remove popper from builds [CUSTOM]

JCavallo committed:
[sao] Add authentication_saml module [PREVIEW] (#148)

Fix #24462

JCavallo committed:
[sao] Fetch real count of records on click [PREVIEW] (#149)

issue11908
Fix #24254

JCavallo committed:
[sao] Patches from merge into 6.4 [CUSTOM]

JCavallo committed:
[sao] Add changes included in the merge from 6.0 [CUSTOM]

JCavallo committed:
[trytond] Restore gitignore [CUSTOM]

JCavallo committed:
[trytond] Check that fields used to order are sortable [PREVIEW]

issue11747
review425751003

JCavallo committed:
[trytond] Use uuid insteadof time to name test db (conflicts) [CUSTOM]

Fix #620

JCavallo committed:
[trytond] Add cache_file_name to activate_modules called from proteus [CUSTOM]

JCavallo committed:
[trytond] Do not crash when parsing fields not in the view [CUSTOM]

JCavallo committed:
[trytond] Bypass translations in reports [CUSTOM]

https://support.coopengo.com/issues/20664

JCavallo committed:
[trytond] Convert dict fields when restoring history [PREVIEW]

JCavallo committed:
[trytond] Remove test on unused view fields [CUSTOM]

JCavallo committed:
[trytond] Disable unused file test [CUSTOM]

JCavallo committed:
[trytond] Add children_definitions to fields_view_get [CUSTOM]

JCavallo committed:
[trytond] Redefine Field properties on Function field [PREVIEW]

In order to use the value from the simulated field, we must redefine them.

issue11759
review425791003

JCavallo committed:
[trytond] Do not init the pool twice in dev mode [CUSTOM]

JCavallo committed:
[trytond] Load static files in werkzeug [CUSTOM]

JCavallo committed:
[trytond] Handle term signals [CUSTOM]

JCavallo committed:
[trytond] Increase access cache sizes [CUSTOM]

JCavallo committed:
[trytond] [6.4-patches] Raise an error when sending an email with missing configuration [CUSTOM] (#214)  (#211)

Fix #PJAZZ-419

JCavallo committed:
[trytond] Add comment on cache invalidation race condition [CUSTOM]

JCavallo committed:
[trytond] trytond: Make convert_domain callable from domain_<field_name> [CUSTOM] (#222)

JCavallo committed:
[trytond] config: Restore missing env variable [CUSTOM]

JCavallo committed:
[trytond] Set the fallback application name when connecting to postgres [PREVIEW] (#224)

PCOOG-15

JCavallo committed:
[trytond] Report contexts get a copy of records with the 'objects' key [CUSTOM] (#225) (#228) (#230)

Fix #PROCK-279

JCavallo committed:
[trytond] Add support for psycopg2cffi and PyPy [CUSTOM] (#231)

Also includes the necessary modification for fallback application name

PCOOG-26

JCavallo committed:
[trytond] Use singleton for TableHandler [PREVIEW] (#227)

* Use singleton for TableHandler [PREVIEW]

* PCOOG-18

* Keep TableHandler signature from previous version

JCavallo committed:
[trytond] trytond: Sorting actions by their decoded names [CUSTOM] (#235)

Fix #PJAZZ-704

JCavallo committed:
[trytond] Reset cache when restore history of ModelSQL [PREVIEW] (#237)

JCavallo committed:
[tryton] Do not display an error dialog on error in cellrenderers [PREVIEW] (#218)

https://bugs.tryton.org/issue11835

JCavallo committed:
[tryton] Trigger group_list_changed to all screens linked to the group [PREVIEW] (#219)

In a tree structure a group and its parent can be linked at any level to a
screen which may have tree views for which the model must be updated.

issue11809
review428041003

Fix #24418

JCavallo committed:
[tryton] Do not filter selection for multiselection fields [PREVIEW] (#220)

issue11857

Fix #24120

JCavallo committed:
[tryton] Fix windows installer encoding [PREVIEW]

JCavallo committed:
[tryton] Remove extra current_record set following merge [TOMERGE]

JCavallo committed:
[tryton] build: Change script to handle force pushes [CUSTOM]

JCavallo committed:
[tryton] Update Master folder [TOMERGE]

JCavallo committed:
[tryton] Remove direct print button [CUSTOM] (#222)

Fix #24365

JCavallo committed:
[tryton] Set the execution level of the standalone app to user [CUSTOM]

Fix #22977

JCavallo committed:
[tryton] Unconnect signals when removing dict entries [PREVIEW]

issue11872
Fix #24431

JCavallo committed:
[tryton] Rebuild languages catalog in build [CUSTOM]

JCavallo committed:
[tryton] Fix _set_value for O2M fields [CUSTOM] (#238)

Fix typo from 6f2735d

JCavallo committed:
[tryton] Ensure that the tree is still realized when getting selected records [PREVIEW]

Fix #24551

JCavallo committed:
[tryton] Fetch real count of records on click [PREVIEW] (#227)

issue11908
review436241003

Fix #24254

JCavallo committed:
[trytond] Add an explicit error message when a reference can't be found when inheriting from a view [CUSTOM] (#240)

Fix #PJAZZ-873

JCavallo committed:
[trytond] Enable the csv extraction of records the user does not have access to [CUSTOM] (#238)

Fix #PJAZZ-586

JCavallo committed:
[trytond] Shorten arguments when logging [PREVIEW] (#241)

* Shorten arguments when logging [PREVIEW]

PROCK-329
https://foss.heptapod.net/tryton/tryton/-/merge_requests/387

JCavallo committed:
[coopengo_modules/debug] debug: Replace the GTK debug object plugin with a wizard in order to use it via SAO (#10)

Fix #PJAZZ-971

JCavallo committed:
[coopengo_modules/debug] debug: Fix a non updated ir.action.keyword record (#11)

Fix #PJAZZ-996

JCavallo committed:
[tryton] Change model on message (#257)

JCavallo committed:
[trytond] Add basic opentelemetry tooling for tracing [CUSTOM] (#236)

Fix PCOOG-56

JCavallo committed:
[trytond] Add migration support from TEXT to JSONB in postgresql [CUSTOM] (#232)

JCavallo committed:
[trytond] Keep using _database_cache if the specific cache hasn't been reset [PREVIEW] (#223)

* Keep using _database_cache if the specific cache hasn't been reset [PREVIEW]

* PCOOG-24

JCavallo committed:
[trytond] Invalidate the table handler's cache only on foreign key change [PREVIEW] (#234)

JCavallo committed:
[trytond] Add authentication_saml module [PREVIEW] (#206)

Fix #24462

JCavallo committed:
[trytond] Don't catch timeout exceptions in error handler (#216)

JCavallo committed:
[trytond] Force named kwargs in activate_modules [CUSTOM] (#207)

Fix #0

JCavallo committed:
[trytond] trytond: remove deprecated bytes_to_wsgi call (#203)

JCavallo committed:
[trytond] trytond: [22.14] Add hook override to allow inactive record modification [CUSTOM] (#210)

Fix #PMETA-81

JCavallo committed:
[modules/account_payment_clearing] account_payment_clearing: Fix test scenario

Fix depending on changes from account_statement PJAZZ-856

JCavallo committed:
[trytond] Ensure that model_id is not None when fetching keyword [CUSTOM] (#243)

PROCK-430

JCavallo committed:
[modules/account] Do not modify reconcile, party_required fields when modifying parent of already saved account [PREVIEW]

issue11871

JCavallo committed:
[modules/account_statement] Remove account move migration [PREVIEW] (#9)

Upstream: https://foss.heptapod.net/tryton/tryton/-/merge_requests/188

JCavallo committed:
[modules/account_statement] account_statement: Prevent the reconciliation of lines with a draft move when validating a statement (#8)

Fix #PJAZZ-856

JCavallo committed:
[modules/account_invoice] Warn when cancelling a move related to an invoice [PREVIEW] (#22)

Fix PCOOG-21

JCavallo committed:
[trytond] Allow users to reset no_update records from xml description [CUSTOM] (#244)

Fix #PJAZZ-840

JCavallo committed:
[trytond] Allow width attribute on buttons while in tree view [CUSTOM]  (#246)

Fix #00000

JCavallo committed:
[sao] Add favicon [CUSTOM]

JCavallo committed:
[sao] Remove extra copy of methods introduced by a7f4aabce [CUSTOM]

JCavallo committed:
[sao] Add Search & Replace to Source widget [CUSTOM] (#157)

PCLAS-186

JCavallo committed:
[tryton] Add authentication_saml module [PREVIEW] (#223)

Fix #24462

JCavallo committed:
[tryton] Ignore missing third part of the version number [CUSTOM]

Fix #24342

JCavallo committed:
[tryton] Do not create specific TrytonAuthenticationError [CUSTOM] (#242)

* Do not create specific TrytonAuthenticationError [CUSTOM]

* Restore UNAUTHORIZED handling in process_exception

JCavallo committed:
[tryton] Fix crash on dictionnary helper in some cases [CUSTOM] (#224)

Fix #24496

JCavallo committed:
[tryton] Force form set_value when switching tree selection in mixed widget [CUSTOM] (#244)

Fix #24611

JCavallo committed:
[tryton] Prevent the resetting of a WizardForm view when the user comes back to it [PREVIEW] (#255)

Fix #PJAZZ-898

JCavallo committed:
[tryton] Add Search & Replace to Source widget [CUSTOM] (#258)

PCLAS-187

JCavallo committed:
[tryton] Reset search offset when the searched text change [PREVIEW] (#259)

CCOOG-43

JCavallo committed:
[tryton] Stop the edition of Selection fields in editable Treeviews on focus-out [PREVIEW] (#266)

PCLAS-215

JCavallo committed:
[tryton] Do not set previous record in selection changed callback when it is destroyed [PREVIEW] (#265)

PJAZZ-1095

JCavallo committed:
[coopengo_modules/debug] debug: Show translated field name and add a filter by field on debug (#13)

Fix #PJAZZ-1051

JCavallo committed:
[tryton] CI

nicoe committed:
Monorepo sync (#1)

* Add missing part of ab3dd964997450a03b7b77a9e52b59e32595c85c [CUSTOM]

* Return a compatible type for multi valued fields in unique_value [PREVIEW]

* Restore lock_signal functionality on group [CUSTOM]

* Fix missing part from 5.2 merge [CUSTOM]

Commit:
Update tryton and pipeline

Commit:
GitLab: Update condition

Commit:
GitLab: fix git command issue

nicoe committed:
Enforce unique value for invisible field [PREVIEW] (#2)

Commit:
GitLab: Fix git command to compare 2 latests tags on current branch

Commit:
GitLab: enable log

Commit:
GitLab: Fix syntax errors

Commit:
update configuration for assume role

Commit:
Terraform: Update volume size + instance type

Commit:
GitLab: Add feature to manage aws assume role

Commit:
GitLab: Fix aws-cli's task for "--duration-seconds" error

Commit:
GitLab: fix token error for aws cli

Commit:
Terraform: remove credit configuration

Commit:
GitLab: Update image for the build step

Commit:
GitLab: centralize pipeline

Commit:
GitLab: Replace project name with GitLab variable

JCavallo committed:
account_payment_clearing: Add a hook on succeed payments method (#11)

nicoe committed:
Robustly handle config file syntax errors [PREVIEW]

JCavallo commented on tryton/tryton/gui/window/dblogin.py:
Au delà du log, ça ne pourrait pas être un message utilisateur ? Sous windows, ils sont à peu près 0% à regarder les logs je pense :)

nicoe commented on tryton/tryton/gui/window/dblogin.py:
Oui on pourrait :), je pense qu'on a un utilitaire qui devrait le permettre.
